lib/puma/cli.rb

```diff
@@ -24,11 +24,10 @@ class CLI
     # Create a new CLI object using +argv+ as the command line
     # arguments.
     #
-    def initialize(argv, log_writer = LogWriter.stdio, events = Events.new)
+    def initialize(argv, log_writer = LogWriter.stdio)
```

Comment:
Not sure we can change the signature of `Puma::CLI`, it is probably considered public API?

Reply:
Yeah, https://github.com/puma/puma/pull/2798 changed the signature and it caused breakage

Reply:
Ok, I am not sure on this if we can change the public API Here. How you want me to take this forward?

The other around is just introduce a `Puma:Events` in `control_cli.rb` and that should also work. instead of introducing it at default config level.